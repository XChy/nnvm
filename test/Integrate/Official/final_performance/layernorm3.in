version https://git-lfs.github.com/spec/v1
oid sha256:7a465e837b7ef705be63798d1b52fd299953d65268b86881ca29487f82594beb
size 5905488
