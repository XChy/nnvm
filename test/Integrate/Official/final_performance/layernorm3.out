-0x1.4765c8p-6
0